Formula/httpd.rb

```diff
@@ -171,6 +171,8 @@ def caveats
       sleep 3
 
       assert_match expected_output, shell_output("curl -s 127.0.0.1:#{port}")
+
+      exec bin/"apxs", "-q", "APU_CONFIG"
```

Comment:
```suggestion
      # Check that `apxs` can find `apu-1-config`.
      system bin/"apxs", "-q", "APU_CONFIG"
```
I don't think `exec` will error on a non-zero exit code. Let's also add a comment so what's happening here is clearer.

Reply:
Done